src/_common/store/app-store.ts

```diff
@@ -34,6 +35,11 @@ export class AppStore extends VuexStore<AppStore, Actions, Mutations> {
 	consents: UserConsents = {};
 	error: number | string | null = null;
 	hasNewStickers = false;
+	timeout: UserTimeout | null = null;
+
+	get isUserTimedOut() {
+		return this.userBootstrapped && !!this.user && !!this.timeout && !this.timeout.isExpired;
```

Comment:
Just making sure, the timeout property is cleared in backend after the timeout is expired rather than returning it with it's past expiration date?

This should be used purely for handling timeouts expiring without a refresh, ye?